src/Installers/Windows/AspNetCoreModule-Setup/ANCMV2/AncmV2Package.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+    <metadata>
+        <id>VS.Redist.Common.AspNetCore.AspNetCoreModuleV2.$MAJOR$.$MINOR$</id>
+        <version>1.0.0</version>
```

Comment:
How is the version modified? The redist packages all push to the same VS feed, so there's going to be conflicts. Or is there some post build magic happening to modify the actual version before publishing?

Reply:
There is some post build magic, though I don't know where it is - the generated .nupkg has this in its .nuspec:

`<version>7.0.0-alpha.1.21621.3</version>`

Reply:
The .nuspec's for our other 2 redist packages have the same thing